src/core/compact_object_test.cc

```diff
@@ -359,4 +363,94 @@ TEST_F(CompactObjectTest, MimallocUnderutilzationWithRealloc) {
   }
 }
 
+TEST_F(CompactObjectTest, JsonTypeTest) {
+  using namespace jsoncons;
```

Comment:
I think we can move to the beginning of the file.

Reply:
OK